.github/workflows/ts_test.yaml

```diff
@@ -71,20 +71,10 @@ jobs:
         working-directory: ethereum_history_api/oracles
         run: yarn test:unit
 
-      - name: Start Anvil
```

Comment:
Idea: Shall we rename file to `e2e_test.yml`?

Reply:
Agree